base-node/index.d.ts

```diff
@@ -350,4 +365,12 @@ export class BaseNode<M extends Meta = Meta> {
    * ```
    */
   destroy (): void
+
+
+  /**
+   * Set headers for current node. 
```

Comment:
We need an example for _all_ methods. Let’s use example with `navigator.language`

Reply:
I've add the example with `navigator`. Added it with `if` because I've checked here is no global object `navigator` in node.